account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/client/SginerClient.java

```diff
@@ -0,0 +1,230 @@
+package com.alchemy.aa.client;
+
+import com.alchemy.aa.Stamper;
+import com.alchemy.aa.Stamper.Stamp;
+import com.alchemy.aa.client.api.AuthJWT;
+import com.alchemy.aa.client.api.AuthUser;
+import com.alchemy.aa.client.api.AuthUser.WhoAmIRequest;
+import com.alchemy.aa.client.api.GetUser;
+import com.alchemy.aa.client.api.SignRawMessage.SignParamter;
+import com.alchemy.aa.client.api.SignRawMessage.SignedRequest;
+import com.alchemy.aa.client.api.SignRawMessage.SignedResponse;
+import com.alchemy.aa.client.api.SignRawMessage.SigningBody;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.google.crypto.tink.util.Bytes;
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.security.GeneralSecurityException;
+import java.time.Instant;
+import java.util.Optional;
+import lombok.Builder;
+import lombok.Getter;
+import org.bouncycastle.util.encoders.Hex;
+
+public class SginerClient {
+
+    public class HttpConfig {
+
+        public HttpConfig(String api_key) {
+            this.api_key = api_key;
+        };
+
+        @Getter
+        private String api_key;
+        @Getter
+        private static String ALCHEMY_URL = "https://api.alchemy.com/v1/";
+    }
+
+    @Builder
+    public record User(String email, String userId, String orgId, String address, String solanaAddress) {
+    }
+
+    @Getter
+    public enum PathName {
+        LOOKUP("lookup"), AUTH("auth"), AUTH_JWT("auth_jwt"), WHOAMI("whoami"), SIGN_PAYLOAD("sign-payload");
+
+        private final String name;
+
+        PathName(String name) {
+            this.name = name;
+        }
+    }
+
+    public enum SigningMode {
+        ETHEREUM, SOLANA
+    }
+
+    private String apiKey;
+
+    class AlchemyHttpClient {
+
+        public AlchemyHttpClient(Stamper stamper, HttpConfig httpConfig) {
+            this.httpConfig = httpConfig;
+            this.stamper = stamper;
+            this.mapper = new ObjectMapper();
+        }
```

Comment:
we should probably support creating a stamper without passing in a stamper (ie. by being able to pass in a TEK key directly)